gtfs/gtfs_utils/gtfs_utils/configuration.py

```diff
@@ -109,9 +111,26 @@ def dict_to_dataclass(data_dict: Dict, data_class: type) -> Dict:
     return data_class_instance
 
 
+def validate_configuration_schema(configuration_dict: dict) -> bool:
+    """
+    Validate the configuration dict against the configuration JSON Schema
+    :param configuration_dict: a dict of the configuration, as loaded with json.load
+    :return: True or raises an error (jsonschema.exceptions.ValidationError)
+    """
+    with open(CONFIGURATION_SCHEMA_FILE_PATH, 'r') as schema_file:
+        config_schema = json.load(schema_file)
+        # The definitions are separated to a different key,
+        # so it won't appear in the sphinx generated docs
```

Comment:
Not really, I'm doing that just because I want to use the sphinx extension and I wan to control the documentation. Another option would be to separate the definitions into a different file, but I thought it would be to complicated.
That's the reason I must rejoin the json for the validation to work.

You can see my conversation with the creator of the package here -
https://github.com/lnoor/sphinx-jsonschema/issues/23